### PR TITLE
Attempt to convert to TypeScript

### PR DESCRIPTION
**Pull request by jgowdy**

### CODE REVIEW

**Review by tarkatronic — APPROVED** [review]
Good enough for government work! 👍🏻

File: package.json [hunk 1]
```diff
@@ -1,16 +1,29 @@
...
-  "type": "module",
+  "main": "dist/index.js",
   "scripts": {
+    "build": "npx tsc",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "Jeremiah Gowdy",
   "license": "MIT",
+  "files": [
+    "dist/index.d.ts"
```
Comment (asilvas) [inline package.json:13]:
```
"dist",
"src"
```

File: package.json [hunk 2]
```diff
@@ -1,16 +1,29 @@
 {
   "name": "cobhan",
-  "version": "1.0.15",
+  "version": "1.0.22",
   "description": "Functions to enable cobhan wrapper libraries",
-  "main": "index.js",
-  "type": "module",
+  "main": "dist/index.js",
```
Comment (asilvas) [inline package.json:5]: `"main": "dist"`

File: package.json [hunk 3]
```diff
@@ -1,16 +1,29 @@
...
+  "devDependencies": {
+    "@types/ffi-napi": "^4.0.5",
+    "@types/node": "^17.0.21",
+    "@typescript-eslint/eslint-plugin": "^5.13.0",
+    "@typescript-eslint/parser": "^5.13.0",
+    "eslint": "^8.10.0",
+    "eslint-config-google": "^0.14.0",
+    "typescript": "^4.6.2"
+  },
+  "types": "dist/index.d.ts"
```
Comment (asilvas) [inline package.json:28]: `"types": "src"`

**Pull request merged**